### PR TITLE
feat(website): add chaindrop to ecosystem page

### LINKED ISSUE #14305: Add Chaindrop Faucet to the ecosystem
### Title of your project

Chaindrop Faucet

### Short (1-2 sentence) description of your project

ChainDrop offers an effortless way to access Web3 test tokens. With just one click, you can receive free Web3 test tokens directly into your wallet.

### Link to your project

chaindrop.org

### Live on Testnet

Yes (default)

### If your project is live on one of Taiko's testnets, please provide evidence of that here (eg. links, screenshots).

https://chaindrop.org/?chainid=167005&token=0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

### Upload a quality logo to your project

Logo

### PR DESCRIPTION
**Pull request by 2manslkh**
Closes #14305

### CODE REVIEW

**Comment by d1onys1us** [comment]
@2manslkh can we take this out of draft? also, do they have a more square / circular icon, would look better for them.

File: packages/website/components/Ecosystem/EcosystemSection.tsx [hunk 1]
```diff
@@ -57,6 +57,15 @@ const ecosystemData: EcosystemData[] = [
     filters: [],
     isLive: true,
   },
+  {
+    icon: "/images/ecosystem/chaindrop.png",
+    name: "Chaindrop Faucet",
```
Comment (d1onys1us) [inline packages/website/components/Ecosystem/EcosystemSection.tsx:62]: this should be above crypton to maintain the alphabetical ordering.

File: packages/website/components/Ecosystem/EcosystemSection.tsx [hunk 2]
```diff
@@ -57,6 +57,15 @@ const ecosystemData: EcosystemData[] = [
...
+    link: "chaindrop.org",
```
Comment (d1onys1us) [inline packages/website/components/Ecosystem/EcosystemSection.tsx:63]: this path is broken, i think `https://chaindrop.org` will work. we parse this string to make it look pretty, but we need the valid href link.

**Review by dionysuzx — CHANGES_REQUESTED** [review]
left some comments

**Comment by 2manslkh** [comment]
Chaindrop is already added in another PR

**Pull request closed without merging**